internal/oauthex/resource_meta.go

```diff
@@ -167,7 +168,7 @@ func getPRM(ctx context.Context, purl string, c *http.Client, wantResource strin
 	if err != nil {
 		return nil, err
 	}
-	// Validate the Resource field to thwart impersonation attacks (section 3.3).
+	// Validate the Resource field (see RFC 9728, section 3.3) as well as issue #560
```

Comment:
remove reference to issue #560 and end with a period

Reply:
done